esp-client/src/main.cpp

```diff
@@ -22,5 +38,63 @@ void loop()
 	{
 		// TODO: Show Output RGB
 	}
+	else 
+	{
+		reconnect();
+	}
+	client.loop();
+}
+void callback(char* topic, byte* message, unsigned int length)
+{
+  	String messageTemp;
+	String topicStr(topic);
+
+	Serial.print("Message arrived on topic: '");
+  	Serial.print(topic);
+  	Serial.print("' Message: '");
+  
+  	for (int i = 0; i < length; i++) {
+    	Serial.print((char)message[i]);
+    	messageTemp += (char)message[i];
+  	}
+  	Serial.println("'");
+
+	if (topicStr == ("/" + MAC))
+	{
+		// TODO: Handle case
+	}
+	else if(topicStr == "/server/online")
+	{
+		// TODO: Let user know
+		if (messageTemp != "connected" && messageTemp != "disconnected") 
+			return;
+		connectedWithNode = messageTemp == "connected";
```

Comment:
Müsste man eventuell in die obere _if_ verschieben, da erhält man die Antwort vom Server, dass das Gerät vom Server erkannt worden ist und hinzugefügt wurde.
Hier erhält man nur, ob der Server online ist. Das könnte man für später in eine `isServerOnline` Variable speichern